build.gradle

```diff
@@ -39,12 +39,17 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     testCompile group: 'commons-io', name: 'commons-io', version: '2.6'
 }
-
+import org.gradle.util.GradleVersion
 task integrationTest(type: Test, group: "verification") {
     inputs.files fileTree(dir: "$project.projectDir/src/integrationTest", exclude: "**/.gradle")
     testClassesDirs = sourceSets.integrationTest.output.classesDirs
     classpath = sourceSets.integrationTest.runtimeClasspath
     mustRunAfter(test)
+    doFirst {
+        if (gradle.startParameter.offline) {
+            systemProperty "eu.xenit.gradle.alfrescosdk.integration.useGradleVersion", GradleVersion.current().version
```

Comment:
hacky, hacky, working on the train ? :-)

Reply:
Yes, when I'm connected via tethering, I don't want to automatically download a lot of different Gradle versions to run the integration tests, even when the `--offline` parameter is used. It should be okay to just run one version in that case.
It's a flag that needs to be passed explicitly to Gradle anyways, so it should be somewhat okay to do this, IMO.